deploy-0chain/action.yml

```diff
@@ -176,61 +170,126 @@ runs:
         helm upgrade --install 0box --wait --set zbox.hostName=${{ env.NAMESPACE }} --set zbox.host=devnet-0chain.net --set zbox.config.blockWorker=https://${{ env.NAMESPACE }}.devnet-0chain.net/dns --set zbox.image.tag=${{ env.ZBOX_TAG }} -n ${{ env.NAMESPACE }}  0chain-helm/zbox --kubeconfig ./kube/${{ env.NAMESPACE }}-config
         helm upgrade --install 0search --set zsearch.hostName=${{ env.NAMESPACE }} --set zsearch.host=devnet-0chain.net --set zsearch.image.tag=${{ env.ZSEARCH_TAG }} -n ${{ env.NAMESPACE }}  0chain-helm/zsearch --kubeconfig ./kube/${{ env.NAMESPACE }}-config
-        helm upgrade --install blobber-stake --wait -n ${{ env.NAMESPACE }} --set blobberStake.config.blockWorker=https://${{ env.NAMESPACE }}.devnet-0chain.net/dns --set blobberStake.blobberCount=6 --set blobberStake.hostName=${{ env.NAMESPACE }} --set blobberStake.host=devnet-0chain.net --set blobberStake.image.tag=${{ env.BLOBBER_STAKE_TAG }} 0chain-helm/blobberStake --kubeconfig ./kube/${{ env.NAMESPACE }}-config
         helm upgrade --install authorizer -n ${{ env.NAMESPACE }} --set authorizer.authorizerCount=3 --set authorizer.hostName=${{ env.NAMESPACE }} --set authorizer.host=devnet-0chain.net --set authorizer.image.tag=${{ env.AUTHORIZER_TAG }} --set authorizer.config.blockWorker=https://${{ env.NAMESPACE }}.devnet-0chain.net/dns 0chain-helm/authorizer --kubeconfig ./kube/${{ env.NAMESPACE }}-config
         ' > ./deploy.sh;
         chmod 777 ./deploy.sh;
         ./deploy.sh
 
-    - name: "Wait for 0Chain network deployment to complete"
+    - name: "Checkout 0wallet CLI"
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.ZWALLET_BRANCH }}
+        repository: 0chain/zwalletcli
+        fetch-depth: 1
+        path: ./zwalletcli
+
+    - name: "Checkout 0box CLI"
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.ZBOX_BRANCH }}
+        repository: 0chain/zboxcli
+        fetch-depth: 1
+        path: ./zboxcli
+
+    - name: "Setup Go"
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.2
+
+    - name: "Build CLI Binaries"
       shell: 'script --return --quiet --command "bash {0}"'
       run: |
-        cd ..
-        echo '#!/bin/bash
-
-        RETRY_ATTEMPT=$1
-        ANNOTATION_LEVEL="warning"
-        if [ "$1" == "final" ]; then
-            ANNOTATION_LEVEL="error"
+        echo
+        echo "======================================================"
+        echo "BUILDING 0WALLET AND 0BOX CLI BINARIES:"
+        echo "======================================================"
+        echo
+        echo "Building zwalletcli [${{ env.ZWALLET_BRANCH }}]..."
+        cd zwalletcli
+        if [[ -n "${{ inputs.custom_go_sdk_version }}" && "${{ inputs.custom_go_sdk_version }}" != "NONE" ]];
```

Comment:
you'll need to add these as inputs to this action now

Reply:
Done